handlers/serviceset.go

```diff
@@ -281,5 +306,6 @@ func (s *ServiceSet) Handler() http.Handler {
 		m.Handle(ArcGISRoot, http.NotFoundHandler())
 	}
 
+	m.HandleFunc("/reload", s.reloadEndpointHandler)
```

Comment:
do we add this handler based on something being enabled or do we check the logic inside the func?

Reply:
inside the function we check if it's enabled!